CONTRIBUTING.md

```diff
@@ -13,3 +13,30 @@ the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/pub
 All contributions to this project will be released under the CC0
 dedication. By submitting a pull request, you are agreeing to comply
 with this waiver of copyright interest.
+
+## Starting Work on an Issue
+
+Issues that are marked with the `ready` label are ripe for the picking!  Simply assign yourself to the issue to and change its label to `in progress` to indicate that you are working on it.
+
+If the issue involves writing code or producing some other change that will result in a pull request, begin by creating yourself a branch with a short descriptive name of the work that includes the issue number at the end, e.g., `document-pr-process-#36` - if you are not a part of the 18F Team, please fork the repository first and then create a branch for yourself with the same convention.
```

Comment:
Ah wait, I see, thought I lost some modifications I had made.  In any case, yes, this is written for outside contributors as well.

I do mention forking the repo if you are outside of the 18F team at the end of that paragraph on line 21, but perhaps this needs to be called out more explicitly, it seems easy to miss.